ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -56,6 +58,8 @@
     private Priority priority;
     private boolean sync;
     private boolean force;
+    private boolean discard;
+    private int bufferSize;
```

Comment:
Just assume a standard size for this.  We don't really need to expose this to the user